packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -1110,6 +1149,13 @@ class _InkResponseState extends State<_InkResponseStateWidget>
     }
   }
 
+  void handleSecondaryTap() {
```

Comment:
I think we should also handle and expose secondary tap cancel in a similar way to  other callbacks and `handleTapCancel` since it is possible a tap is cancelled and we want the splash to cancel in that case.

Reply:
Hey, @Renzo-Olivares I've updated the PR, sorry for the delay missed the notifications.